source/Integers/Abs.lagda

```diff
@@ -169,3 +171,29 @@ abs-over-mult' (negsucc x) (negsucc y) = I
     iii = pos-multiplication-equiv-to-ℕ (succ x) (succ y) ⁻¹
 
 \end{code}
+
+Lane Biocini 2023
+
+In this section I prove a convenience lemma about the Absolute Value operation,
+then go on to prove a lemma regarding the equivalence of the addition of a positive
+and negative Integer to the Absolute Difference operation in the Naturals, which
+will help us when we prove the triangle inequality in the Integers.
+
+\begin{code}
+
+pos-abs-is-absℤ : (x : ℤ) → pos (abs x) ＝ absℤ x
+pos-abs-is-absℤ (pos x) = refl
+pos-abs-is-absℤ (negsucc x) = refl
+
+abs-pos-plus-negsucc : (x y : ℕ) → abs (pos x +negsucc y) ＝ ∣ x - succ y ∣
+abs-pos-plus-negsucc zero y = ap abs (ℤ+-comm (pos 0) (negsucc y))
+abs-pos-plus-negsucc (succ x) zero = refl
+abs-pos-plus-negsucc (succ x) (succ y) =
```

Comment:
I will approve this but will have to fix the indentation later. In `TypeTopology` the convention is that indentation is by one space character.

Reply:
(Please fix all instances of that.)